Automattic/jetpack
# VideoPress: add Cancel button to uploader component

This PR adds a Cancel button when the video is replaced by uploading a new file.

<img width="632" alt="Screen Shot 2023-01-05 at 18 04 00" src="https://user-images.githubusercontent.com/77539/210849563-c3788871-bc90-4e1a-a129-26bc64f82444.png">

Fixes #

#### Changes proposed in this Pull Request:


* VideoPress: add Cancel button to uploader component

#### Other information:



#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the block editor
* Create/edit a video block
* Replace the video by uploading a new file
* Confirm you see the new `Cancel` button
* Confirm you cancel the replacement process by clicking on the Cancel button
* Confirm that if you try to replace the file with the same file uploaded before, it resumes uploading.
* Confirm the video is replaced once the video is uploaded

https://user-images.githubusercontent.com/77539/210848825-b0071930-6c57-4fc0-b907-b68366f5f5c8.mov
